test/fitnesse/testsystems/slim/SlimTestSystemTableProcessingTest.java

```diff
@@ -322,7 +340,9 @@ public void kill() {
   private static class IgnoreOnNullPassOtherwiseSlimExpectation implements SlimExpectation {
     @Override
     public TestResult evaluateExpectation(Object returnValues) {
-      return new SlimTestResult(null == returnValues ? ExecutionResult.IGNORE : ExecutionResult.PASS);
+//      if(returnValues.toString().contains("IGNORE_SCRIPT_TEST"))
```

Comment:
Please remove lines instead of commenting out

Reply:
Thanks